frontend/src/components/applyForm/types.ts

```diff
@@ -40,11 +40,22 @@ export type FieldErrors = ErrorObject<
   unknown
 >[];
 
+export type WidgetTypes = "Checkbox" | "Text" | "TextArea" | "Radio" | "Select";
+
 export type UiSchemaField = {
   type: "field";
-  definition?: `/properties/${string}`;
-  schema?: SchemaField;
-};
+  widget?: WidgetTypes;
```

Comment:
Added optional `widget` but also improved type checking for the ui schema. The field must have `definition`, `schema`, or both.